common/src/main/java/io/netty/util/AsyncMapping.java

```diff
@@ -18,11 +18,11 @@
 import io.netty.util.concurrent.Future;
 import io.netty.util.concurrent.Promise;
 
-public interface AsyncMapping<IN, OUT> {
+public interface AsyncMapping<U, R> {
```

Comment:
can you explain the motivation for this change? `U` doesn't seem very clear to me.